### DISCUSSION TITLE
Midi (pichbend data) crashing windows

### DISCUSSION THREAD

**Discussion by josehdx** [tttapa/Control-Surface#863] [Q&A] [answered]
Dear Dev

I think my issue is happening only on windows (i read similar post of PB crashing some PCs) but when i set the Potentiometer resolution above 5, and after doing a quick press and release of the know  to my spring loaded potentiometer, i have been crashing all the MIDI related apps (even midi monitors) on my PC (connected to the arduino board via BLE), I have mentioned on previous post that i also use and android app called "Piano MIDI BLE" which i used for testing my MIDI messages, When i do the same test, i dont have any problem. So i wonder if you can give me a hint about a way to improve this. I know the obvius  solutionc will be to decrease the resolution but this makes the steps increment bigger than 1% (considering a full run of 0 to 100) so i would like to know your opinion about this issue, Thanks!

**Comment by tttapa** [comment, marked as answer]
A simple workaround would be to add a small delay in your main loop, so Windows doesn't get overwhelmed by a high rate of MIDI messages.

Which board are you using?

**Reply by josehdx** [reply]
Ok i think i saw such workaround before on the internet but I didn't think about using it. Thanks.

In am using the esp32 - s3

Regards!

**Comment by josehdx** [comment]
Solved with delay (1)

Thanks
